web/components/TextBlock/Tickets/Tickets.tsx

```diff
@@ -27,23 +30,59 @@ export const Tickets = ({ value: { type, tickets } }: TicketsProps) => {
   const includedTypes = Array.from(
     new Set(tickets.map((ticket) => ticket.included).flat())
   );
+
+  const currentPriceAndAvailability = (ticket: Ticket) =>
+    ticket.priceAndAvailability?.reduce((acc, current) => {
+      const fromDate = parseISO(current.from);
+      const currentDate = new Date();
+      // Check if current date is before the from date
+      if (compareAsc(currentDate, fromDate) == 1) {
+        return current;
+      }
+      return acc;
+    }, ticket.priceAndAvailability[0]);
+
   return (
     <GridWrapper>
       <table className={styles.table}>
         <thead>
           <tr>
             <th />
-            {tickets.map((ticket) => (
-              <th key={ticket._id} scope="col" className={styles.ticketInfo}>
-                <div className={styles.name}>{ticket.type}</div>
-                <dl className={styles.priceList}>
-                  <dt className={styles.priceLabel}>Price</dt>
-                  <dd className={styles.price}>
-                    {ticket.price ? `$${ticket.price}` : 'Free'}
-                  </dd>
-                </dl>
-              </th>
-            ))}
+            {tickets.map((ticket) => {
+              const currentPrice = currentPriceAndAvailability(ticket);
+              return (
+                <th key={ticket._id} scope="col" className={styles.ticketInfo}>
+                  <div className={styles.name}>{ticket.type}</div>
+                  {ticket.description && (
+                    <div className={styles.description}>
+                      {ticket.description.map((value) => (
+                        <Block key={value._key} value={value} />
```

Comment:
This is really a part of a more "global" issue (related to the `_span`/`<span>` thing discussed on Slack), so I consider it a nit here.

But it seems to me that we neither support nor need to support any advanced kinds of blocks here, like `RichText` or `Venue`. So it might make sense to render the value more directly.

I wonder if we should use `<PortableText>` directly in cases such as this, or if there should be a simplified version of the `TextBlock.tsx` thing. And I also suspect that the `Block` component means that all inline formatting might be lost, because it doesn't seem to use `<PortableText>` itself.

Reply:
Using `<PortableText />` directly here makes sense here, and also looks cleaner. 8fc8688

(I don't even remember what or why `<Block />` is at this point, a small round of cleanup is due after launch. :sweat_smile:)